src/ua_util.c

```diff
@@ -225,17 +225,27 @@ UA_ByteString_fromBase64(UA_ByteString *bs,
     return UA_STATUSCODE_GOOD;
 }
 
+const UA_KeyValueMap UA_KEYVALUEMAP_NULL = {0};
+
+UA_KeyValueMap *
+UA_KeyValueMap_new(size_t sizeHint) {
+    UA_KeyValueMap *kvm = (UA_KeyValueMap*) UA_calloc(sizeof(UA_KeyValueMap), 1);
```

Comment:
Always check for malloc failure.
That happens on embedded and people actively ask for it.
Doesn't matter now if _new is a naked calloc.